packages/preset-create-react-app/index.js

```diff
@@ -0,0 +1,77 @@
+const path = require('path');
+const { logger } = require('@storybook/node-logger');
+const mergePlugins = require('./helpers/mergePlugins');
+const reactScriptsPath = require('./helpers/reactScriptsPath');
```

Comment:
should this be a function instead?

Reply:
I thought about it, but it felt like this solution would be suffice in that you're just importing a path name (if found) - and it's fast in that it only looks once. A function would run multiple times (as we use it multiple times), and then we'd need to memoise.

Reply:
I did it. But it's invoked once only.